modules/shortcodes/vimeo.php

```diff
@@ -78,7 +78,22 @@ function vimeo_shortcode( $atts ) {
 		$height = round( ( $width / 640 ) * 360 );
 	}
 	
+	/**
+	 * Filter the Vimeo player width.
+	 *
+	 * @since 3.4-beta1
```

Comment:
Your inline docs are already part of `master` though, so they won't be visible anywhere in our docs until they make it into a stable release.